### PR TITLE
Add Slice to FusionTranslation

### PR DESCRIPTION
**Pull request by rdspring1**
This PR adds support for `Slice` operations to `FusionTranslation`.

### CODE REVIEW

**Comment by rdspring1** [comment]
!build

**Comment by rdspring1** [comment]
!build

File: csrc/python_frontend/translation.cpp [hunk 1]
```diff
@@ -758,6 +758,39 @@ class FusionTranslator : public OptInConstDispatch {
         {fd_->recordingState(output())}));
   }
 
+  // Map SliceOp to python frontend
+  void handle(const SliceOp* sop) final {
+    const std::vector<nvfuser::Slice>& slices = sop->getRanges();
```
Comment (jacobhinkle) [inline csrc/python_frontend/translation.cpp:763]: It looks like `getRanges` does not return a reference.

**Review by jacobhinkle — APPROVED** [review]

**Comment by rdspring1** [comment]
!build

**Pull request merged**